logstash-core/spec/api/lib/api/support/resource_dsl_methods.rb

```diff
@@ -14,7 +14,13 @@ def hash_to_mapping(h, path=[], mapping={})
 
   def test_api(expected, path)
     context "GET #{path}" do
-      let(:payload) { LogStash::Json.load(last_response.body) }
+      let(:payload) do
+        begin
+          LogStash::Json.load(last_response.body)
+        rescue LogStash::Json::ParserError => e
+          require 'pry'; binding.pry
```

Comment:
pry when something wrong happens? I think this might be good for debugging but can cause unnecessary problems if raised unintentional